### PR TITLE
completed the task

### PR DESCRIPTION
**Pull request by Yehorkk**
[DEMO LINK](https://yehorkk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://yehorkk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,32 @@
...
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+
+    </header>
+    <!--<h1>Moyo header</h1> -->
```
Comment (denys-danyliuk) [inline src/index.html:35]: delete comments

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,32 @@
...
+            data-qa="hover">Laptops & Computers</a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+
+    </header>
```
Comment (denys-danyliuk) [inline src/index.html:34]: don't add empty lines between child and parent elements, check code style

File: src/index.html [hunk 3]
```diff
@@ -6,9 +6,32 @@
...
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img src="images/logo.png" alt="moyo"
+        class="logo__image">
+      </a>
+      <nav class="menu">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link is-active">Apple</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
```
Comment (denys-danyliuk) [inline src/index.html:21]:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a href="#" class="nav__link">
              Samsung
            </a>
          </li>
```

File: src/index.html [hunk 4]
```diff
@@ -6,9 +6,32 @@
...
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```
Comment (denys-danyliuk) [inline src/index.html:25]: each attribute should be on a new line when there is more than 2 of them, check code style

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: black;
+  text-decoration: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: var(--h);
```
Comment (denys-danyliuk) [inline src/style.css:37]: don't hardcode exact height, use paddings

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 6]
```diff
@@ -6,9 +6,68 @@
...
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="hover">Laptops & Computers</a>
```
Comment (etojeDenys) [inline src/index.html:42]:
```suggestion
            <a 
              href="#" 
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,71 @@
...
+  margin-left: 20px;
+}
+
+.nav__link:hover {
+  color: var(--c);
+}
+
+.is-active {
+  color: var(--c);
+}
```
Comment (etojeDenys) [inline src/style.css:60]: we can join it using the comma

**Review by etojeDenys — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 8]
```diff
@@ -6,9 +6,69 @@
...
     <title>Moyo header</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
+    <header>
+      <a href="#home" class="logo">
+        <img src="images/logo.png"
+             alt="moyo"
+             class="logo__image">
```
Comment (anastasiiavorobiova) [inline src/index.html:17]:
```suggestion
        <img 
          src="images/logo.png"
          alt="moyo"
          class="logo__image"
        >
```
Consider following the code style guide

File: src/index.html [hunk 9]
```diff
@@ -6,9 +6,69 @@
...
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#"
+               class="nav__link"
+               data-qa="hover">Laptops & Computers</a>
```
Comment (anastasiiavorobiova) [inline src/index.html:43]:
```suggestion
            <a 
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```
The same issue here

File: src/index.html [hunk 10]
```diff
@@ -6,9 +6,69 @@
...
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
     <h1>Moyo header</h1>
```
Comment (anastasiiavorobiova) [inline src/index.html:72]:
```suggestion

```

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,68 @@
+:root {
+  --h: 60px;
+  --c: #00acdc;
```
Comment (anastasiiavorobiova) [inline src/style.css:3]: Consider using meaningful names for variables, for example '--accent-color', etc.

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

**Pull request closed without merging**